src/shogun/base/SGObject.cpp

```diff
@@ -741,3 +776,29 @@ CSGObject* CSGObject::clone()
 	SG_DEBUG("leaving %s::clone(): Clone successful\n", get_name());
 	return copy;
 }
+
+Any CSGObject::_set(BaseTag& _tag, const Any& any)
+{
+	return self->set(_tag, any);
+}
+
+Any CSGObject::_get(BaseTag& _tag) const
+{
+	Any any = self->get(_tag);
+	if(any.empty())
+	{
+		SG_ERROR("There is no parameter called \"%s\" in %s",
+			_tag.name().c_str(), get_name());
```

Comment:
Can you put null checks here to avoid segfaults?
Or add a comment why segfaults cannot happen?